pyk/src/pyk/ktool/kompile.py

```diff
@@ -512,6 +512,7 @@ class KompileArgs:
     coverage: bool
     bison_lists: bool
     outer_parsed_json: bool
+    ignore_warnings: Iterable[str] | None
```

Comment:
This would be better as an argument in `Kompile.__call__`, please move it there.

Reply:
Thank you! Just did so. Let me know if anything else needs changing!